modules/cnf-about-collecting-ptp-data.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * networking/using-ptp.adoc
+
+:_content-type: PROCEDURE
+[id="cnf-about-collecting-nro-data_{context}"]
+= Collecting PTP Operator data
```

Comment:
Should this be 
```suggestion
= Collecting Precision Time Protocol (PTP) Operator data
```
or is it just PTP when paired with Operator? I see it both ways in the docs